FirebaseAdmin/FirebaseAdmin/AppCheck/AppCheckTokenVerify.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FirebaseAdmin.Auth;
+using FirebaseAdmin.Auth.Jwt;
+using Google.Apis.Auth;
+using Google.Apis.Util;
+
+namespace FirebaseAdmin.Check
+{
+    internal class AppCheckTokenVerify
+    {
+        /*private const string IdTokenCertUrl = "https://www.googleapis.com/robot/v1/metadata/x509/"
```

Comment:
Why is this here? If we don't use it why don't we remove it?

Reply:
I have updated the AppCheckTokenVerifier.cs file, and the issue has been resolved.